src/notificationarea.cpp

```diff
@@ -77,7 +78,20 @@ void NotificationArea::setHeight(int contentHeight)
     // FIXME: Qt does not seem to update QDesktopWidget::primaryScreen().
     // After we change the primary screen with xrandr, Qt still returns the same value.
     // I think it's a bug of Qt.
-    QRect workArea = qApp->desktop()->availableGeometry(qApp->desktop()->primaryScreen());
+
+    QRect workArea{};
+    const auto screens = qApp->screens();
```

Comment:
`screens` isn't needed outside the block that follows its definition. Please put it inside the block.

Reply:
Done.